src/main/java/gregtech/common/metatileentities/multi/MetaTileEntitySpentFuelPool.java

```diff
@@ -0,0 +1,99 @@
+package gregtech.common.metatileentities.multi;
+
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.api.metatileentity.multiblock.IMultiblockPart;
+import gregtech.api.metatileentity.multiblock.RecipeMapMultiblockController;
+import gregtech.api.pattern.BlockPattern;
+import gregtech.api.pattern.FactoryBlockPattern;
+import gregtech.api.pattern.PatternMatchContext;
+import gregtech.api.recipes.RecipeMaps;
+import gregtech.client.renderer.ICubeRenderer;
+import gregtech.client.renderer.texture.Textures;
+import gregtech.common.blocks.BlockMetalCasing;
+import gregtech.common.blocks.BlockNuclearCasing;
+import gregtech.common.blocks.MetaBlocks;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.init.Blocks;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+import static gregtech.api.util.RelativeDirection.*;
+
+public class MetaTileEntitySpentFuelPool extends RecipeMapMultiblockController {
+
+    public MetaTileEntitySpentFuelPool(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId, RecipeMaps.SPENT_FUEL_POOL_RECIPES);
+    }
+
+    @Override
+    public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
+        return new MetaTileEntitySpentFuelPool(metaTileEntityId);
+    }
+
+    @Override
+    public boolean hasMaintenanceMechanics() {
```

Comment:
Does this structure need muffler mechanics really either? Not sure if that should also be overridden and return false. If you think there should be a muffler, it should also check for muffler in `autoabilities` down below.

Reply:
The superclass method already automatically returns false, so this is not needed.